performance-tests/bench/legacy_plotting/README

```diff
@@ -1,9 +1,39 @@
 To generate new data and image files based on a scoreboard test run directory:
 
-./generate-test-results.sh <TEST_RUN_DIR>
+1) Set up your environment by sourcing $DDS_ROOT/setenv.sh and $BENCH_ROOT/set_bench_env.sh
 
-which will create "<TEST_RUN_DIR>/data" with relevant data files (including json for website)
+  $ cd OpenDDS
+  $ . setenv.sh
+  $ cd performance-tests/bench
+  $ . set_bench_env.sh
 
-./plot-test_results.sh <TEST_RUN_DIR> <OUTPUT_DIR>
+2) Extract the Bench v1 formatted data files from a Bench v2 test run of the b1_latency_* scenarios (e.g. what gets generated by /testbed/vm/images/centos/7.7/x64/ext/bench/run_b1_scenarios.sh when run by the 'bench' user on the VM 'centipede'). This will create the directory "<TEST_RUN_DIR>/data" with relevant data files (including json for website).
```

Comment:
Where is the default location for data to be placed on Centipede? I assume we want "/testbed/vm/images/centos/7.7/x64/ext/bench/" to stay clean so we place it somewhere else.

Reply:
The script itself uses temporary directories and archives the results in a timestamped test run directory in /www-docs/

Reply:
Which is to say, I don't think we need to add more details about that here